### DISCUSSION TITLE
Many, many thanks!!!

### DISCUSSION THREAD

**Discussion by praeceptum** [valinet/ExplorerPatcher#86] [General]
MANY, MANY THANKS to everyone (valinet!) involved with ExplorerPatcher.

The default Windows 11 UI almost seems designed to _resist_ productivity, especially for developers that need to launch applications quickly and switch between lots of them frequently.  ExplorerPatcher _saves the world_ just by restoring TaskBar toolbars (QuickLaunch).

**Comment by valinet** [comment]
You're welcome. And thank you very much for the kind words. Hopefully, more capabilities can be brought to the app soon.

**Comment by danielsawan** [comment]
100% agree good work guys keep it up !

**Reply by MickeyUK** [reply]
This is fantastic, the new taskbar has been making me tear my hair out. Great work!

**Comment by Deadite** [comment]
Another thank you from me, it's wonderful to have the best of both Windows 10 and 11.  Also OpenShell works perfectly with the restored taskbar so no need to use the cumbersome new start menu.  Feels like having a modern version of Windows 7!

**Reply by valinet** [reply]
Thanks. I am glad that it works well for you.

**Comment by bakertony** [comment]
You guys involved with ExplorerPatcher are legends! Thanks heaps! LOVE having my task bar docked to the left side of my screen again, ungrouped and small icons. 
Have sent a donation.

**Reply by Gaurav-Original-ClassicShellTester** [reply]
It's just one talented and  hard working developer: https://valinet.ro/

**Reply by valinet** [reply]
Thank you both!

**Comment by RussKie** [comment]
Mate, you're a 💯 legend! I can now be productive again.

**Reply by valinet** [reply]
Thank you 🙏
